setup.json

```diff
@@ -53,7 +57,11 @@
 			"name": "DailyPythonScripts",
 			"source": "git+https://github.com/BristolTopGroup/DailyPythonScripts.git",
 			"destination": "",
-			"provider": "pip"
+			"provider": "pip",
+			"setup-cmds": [
+				"cd ../../.. && ln -s ../external/miniconda/envs/ntp/lib/python2.7/site-packages/dps DEV/dps",
```

Comment:
in `CMSSW_SRC` I believe. But you are right, this needs changing!
Since this has to work with PR #257, I will need to update this PR. In order to make both work, I would have to deploy DPS with git (like AS). Please consider the other two PRs first, I will come back to this one.